Makefile

```
@@ -1,5 +1,5 @@
 
-TARGETS := $(shell ls scripts)
+TARGETS := $(shell ls scripts | grep -v dapper-image)
```

In this case you can remove the grep -b dapper-image and L11:L13 right?
```suggestion
TARGETS := $(shell ls scripts)
```

---

I'd keep it for now for consistency with subm since we alrady have a validate target and will also have e2e and maybe some other ones

---

Actually makes sense.. I'll try it

---

But Make knows how to deal with that on its own, without the `grep -v` and the extra rule — it breaks circular dependencies properly...

---

Yeah it seems that it doe, cool, it's gone then :)